ddtrace/contrib/gevent/greenlet.py

```diff
@@ -17,15 +34,23 @@ class TracedGreenlet(gevent.Greenlet):
     ``Greenlet`` class means extending automatically ``TracedGreenlet``.
     """
     def __init__(self, *args, **kwargs):
```

Comment:
This whole `__init__` method should be able to go away if all we are doing it calling `super().__init__()`


```python
class TracedGreenlet(TracingMixin, gevent.Greenlet):
    pass

class TracedIMapUnordered(TracingMixing, gpool.IMapUnorderd):
    pass
```

Reply:
also, not a big deal if we keep these as-is (with `__init__`), not a required change to make to land this.